crates/chain/src/keychain.rs

```diff
@@ -72,6 +67,7 @@ impl<K: Ord> Append for DerivationAdditions<K> {
         self.0.append(&mut other.0);
     }
 
+    /// Returns whether the additions are empty.
     fn is_empty(&self) -> bool {
```

Comment:
The other one was pub, this one is not

Reply:
I saw that but also saw that the Append trait is public.